### PR TITLE
Fix docs links and add examples to sidebar

### PR DESCRIPTION
**Pull request by tastypackets**
## This relates to...

Docs

## Rationale

Links should work

## Changes

- Added examples to sidebar
- Fixed broken links on home page

image

### Features

N/A

### Bug Fixes

N/A

### Breaking Changes and Deprecations

N/A

## Status

- [x] I have read and agreed to the [Developer's Certificate of Origin][cert]
- [S] Tested
- [S] Benchmarked (**optional**)
- [x] Documented
- [x] Review ready
- [ ] In review
- [ ] Merge ready

[cert]: https://github.com/nodejs/undici/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -94,14 +94,14 @@ For more information about their behavior, please reference the body mixin from
 
 ## Common API Methods
 
-This section documents our most commonly used API methods. Additional APIs are documented in their own files within the [documentation](./documentation/) folder and are accessible via the navigation list on the left side of the docs site.
+This section documents our most commonly used API methods. Additional APIs are documented in their own files within the API folder and are accessible via the navigation list on the left side of the docs site.
```
Comment (tastypackets) [inline docs/README.md:97]:
```suggestion
This section documents our most commonly used API methods. Additional APIs are documented in their own files within the [docs](./docs) folder and are accessible via the navigation list on the left side of the docs site.
```

File: docs/README.md [hunk 2]
```diff
@@ -110,14 +110,14 @@ Returns a promise with the result of the `Dispatcher.request` method.
 
 Calls `options.dispatcher.request(options)`.
 
-See [Dispatcher.request](./documentation/docs/api/Dispatcher.md#dispatcherrequestoptions-callback) for more details, and [request examples](./documentation/examples/README.md) for examples.
+See [Dispatcher.request](docs/api/Dispatcher.md#dispatcherrequestoptions-callback) for more details, and [request examples](./examples/README.md) for examples.
```
Comment (Uzlopak) [inline docs/README.md:113]:
```suggestion
See [Dispatcher.request](./docs/api/Dispatcher.md#dispatcherrequestoptions-callback) for more details, and [request examples](./examples/README.md) for examples.
```

File: docs/README.md [hunk 3]
```diff
@@ -110,14 +110,14 @@ Returns a promise with the result of the `Dispatcher.request` method.
...
 
 ### `undici.stream([url, options, ]factory): Promise`
 
 Arguments:
 
 * **url** `string | URL | UrlObject`
-* **options** [`StreamOptions`](./documentation/docs/api/Dispatcher.md#parameter-streamoptions)
+* **options** [`StreamOptions`](docs/api/Dispatcher.md#parameter-streamoptions)
```
Comment (Uzlopak) [inline docs/README.md:120]:
```suggestion
* **options** [`StreamOptions`](./docs/api/Dispatcher.md#parameter-streamoptions)
```

File: docs/README.md [hunk 4]
```diff
@@ -127,14 +127,14 @@ Returns a promise with the result of the `Dispatcher.stream` method.
 
 Calls `options.dispatcher.stream(options, factory)`.
 
-See [Dispatcher.stream](documentation/docs/api/Dispatcher.md#dispatcherstreamoptions-factory-callback) for more details.
+See [Dispatcher.stream](docs/api/Dispatcher.md#dispatcherstreamoptions-factory-callback) for more details.
```
Comment (Uzlopak) [inline docs/README.md:130]:
```suggestion
See [Dispatcher.stream](./docs/api/Dispatcher.md#dispatcherstreamoptions-factory-callback) for more details.
```

File: docs/README.md [hunk 5]
```diff
@@ -127,14 +127,14 @@ Returns a promise with the result of the `Dispatcher.stream` method.
...
 
 ### `undici.pipeline([url, options, ]handler): Duplex`
 
 Arguments:
 
 * **url** `string | URL | UrlObject`
-* **options** [`PipelineOptions`](documentation/docs/api/Dispatcher.md#parameter-pipelineoptions)
+* **options** [`PipelineOptions`](docs/api/Dispatcher.md#parameter-pipelineoptions)
```
Comment (Uzlopak) [inline docs/README.md:137]:
```suggestion
* **options** [`PipelineOptions`](./docs/api/Dispatcher.md#parameter-pipelineoptions)
```

File: docs/README.md [hunk 6]
```diff
@@ -144,7 +144,7 @@ Returns: `stream.Duplex`
 
 Calls `options.dispatch.pipeline(options, handler)`.
 
-See [Dispatcher.pipeline](documentation/docs/api/Dispatcher.md#dispatcherpipelineoptions-handler) for more details.
+See [Dispatcher.pipeline](docs/api/Dispatcher.md#dispatcherpipelineoptions-handler) for more details.
```
Comment (Uzlopak) [inline docs/README.md:147]:
```suggestion
See [Dispatcher.pipeline](./docs/api/Dispatcher.md#dispatcherpipelineoptions-handler) for more details.
```

File: docs/README.md [hunk 7]
```diff
@@ -153,7 +153,7 @@ Starts two-way communications with the requested resource using [HTTP CONNECT](h
 Arguments:
 
 * **url** `string | URL | UrlObject`
-* **options** [`ConnectOptions`](documentation/docs/api/Dispatcher.md#parameter-connectoptions)
+* **options** [`ConnectOptions`](docs/api/Dispatcher.md#parameter-connectoptions)
```
Comment (Uzlopak) [inline docs/README.md:156]:
```suggestion
* **options** [`ConnectOptions`](./docs/api/Dispatcher.md#parameter-connectoptions)
```

File: docs/README.md [hunk 8]
```diff
@@ -162,7 +162,7 @@ Returns a promise with the result of the `Dispatcher.connect` method.
 
 Calls `options.dispatch.connect(options)`.
 
-See [Dispatcher.connect](documentation/docs/api/Dispatcher.md#dispatcherconnectoptions-callback) for more details.
+See [Dispatcher.connect](docs/api/Dispatcher.md#dispatcherconnectoptions-callback) for more details.
```
Comment (Uzlopak) [inline docs/README.md:165]:
```suggestion
See [Dispatcher.connect](./docs/api/Dispatcher.md#dispatcherconnectoptions-callback) for more details.
```

File: docs/README.md [hunk 9]
```diff
@@ -301,7 +301,7 @@ Upgrade to a different protocol. See [MDN - HTTP - Protocol upgrade mechanism](h
 Arguments:
 
 * **url** `string | URL | UrlObject`
-* **options** [`UpgradeOptions`](documentation/docs/api/Dispatcher.md#parameter-upgradeoptions)
+* **options** [`UpgradeOptions`](docs/api/Dispatcher.md#parameter-upgradeoptions)
```
Comment (Uzlopak) [inline docs/README.md:304]:
```suggestion
* **options** [`UpgradeOptions`](./docs/api/Dispatcher.md#parameter-upgradeoptions)
```

File: docs/README.md [hunk 10]
```diff
@@ -310,7 +310,7 @@ Returns a promise with the result of the `Dispatcher.upgrade` method.
 
 Calls `options.dispatcher.upgrade(options)`.
 
-See [Dispatcher.upgrade](documentation/docs/api/Dispatcher.md#dispatcherupgradeoptions-callback) for more details.
+See [Dispatcher.upgrade](docs/api/Dispatcher.md#dispatcherupgradeoptions-callback) for more details.
```
Comment (Uzlopak) [inline docs/README.md:313]:
```suggestion
See [Dispatcher.upgrade](./docs/api/Dispatcher.md#dispatcherupgradeoptions-callback) for more details.
```

**Review by metcoder95 — APPROVED** [review]

**Review by Uzlopak — APPROVED** [review]

File: docs/docsify/sidebar.md [hunk 11]
```diff
@@ -1,6 +1,7 @@
 <!-- Sidebar for Docsify -->
 
 * [**Home**](/ "Node.js Undici")
+* [Examples](/examples/ "Undici Examples")
```
Comment (Uzlopak) [inline docs/docsify/sidebar.md:4]: This looks like this

image

Should we put the examples maybe into best practise?

**Comment by Uzlopak** [comment]
So?

**Review by Ethan-Arrowood — APPROVED** [review]

**Comment by Uzlopak** [comment]
@Ethan-Arrowood Can we move the examples link to a much less dominant position?

**Comment by Ethan-Arrowood** [comment]
Doesn't matter to me one way or another. I'm hoping to change our doc site sooner than later anyways.

**Review by Uzlopak — APPROVED** [review]
@Ethan-Arrowood 
Can you approve and merge please?

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**